components/NavBar.vue

```diff
@@ -1,6 +1,11 @@
 <template>
   <nav>
-    <nav :style="{visibility: visible, opacity: opacity}" class="navbar" role="navigation" aria-label="main navigation">
+    <nav
+      :style="{visibility: visible, opacity: opacity}"
+      class="navbar"
+      role="navigation"
+      aria-label="main navigation"
+    >
```

Comment:
I think that design wanted to use the sign post as the nav bar just for the hero. The scrolling is still good for the rest of the site though, so can you just set its visibility to hidden initially. 

This is what is on the figma, it should get bigger when hovered to make it more obvious that it is a nav.

![image](https://user-images.githubusercontent.com/52933714/90304830-f0771800-de78-11ea-825d-34c792742e1b.png)

Reply:
@rmehri01 I see! I'm not sure what's the best way to go about implementing this part though, do you have any suggestion? My initial thought is to separate this part from the background, but not sure how to properly place it separately

Reply:
@alice-0-kim Yeah the sign definitely has to be separated from the background with each section (About, FAQ, Sponsors) being it's own thing. I'm not entirely sure how to place them on the post in the background and make it stay there when scaled on different screen sizes, I can look into it.

Reply:
Do you know if absolute/fixed/relative positioning would work?

Reply:
I'd imagine it would still require some sort of a way to locate its position relative to the background, but it would have to be adjusted when the screen resizes. Should we ask around see if anyone has encountered similar issues before?

Reply:
Yeah, that is a good idea.

Reply:
Per Jaccy's comment, I looked into last year's nwHacks repo and how they dealt with the similar issue, and it seems like [they did it with padding](https://nwplus2020-2021.slack.com/archives/CV9APJ63X/p1597630654188500). Should we do the same?

Reply:
Oh yeah I forgot that was on the nwHacks website, I'll look into how that is done too and see.

Reply:
@alice-0-kim Also I think that's the wrong repo, that's nwHacks 2019.

Reply:
Ahh yes, [here](https://github.com/nwplus/nwhacks2020/blob/6ff4766e78e458243a716a4d9f66e707c7e8bae5/components/Intro.vue#L117)'s the correct reference to the nwHacks 2020 repo and how they used the padding.